plugins/nto/skills/scripts/analyze_node_tuning.py

```
@@ -0,0 +1,895 @@
+        ]
+        for candidate in candidates:
+            if candidate.exists():
+                ps_snapshot = candidate
+                break
+    if ps_snapshot is None:
+        return {"top_processes": [], "recommendations": []}
+    text = _safe_read_text(ps_snapshot)
+    if not text:
+        return {"top_processes": [], "recommendations": []}
+    lines = [line for line in text.splitlines() if line.strip()]
+    header = lines[0]
```

Ruff (0.14.4)

Local variable `header` is assigned to but never used

Remove assignment to unused variable `header`

(F841)